gnat2goto/driver/tree_walk.adb

```diff
@@ -994,6 +994,11 @@ package body Tree_Walk is
             --  but I can't think of one without delving deeply, and I
             --  think such uses would be unusual (TJJ 21/11/2019)
             null;
+         when N_Generic_Subprogram_Declaration
+           | N_Generic_Package_Declaration =>
+            --  Generic subprograms and packages
+            --  don't need to specially handled
```

Comment:
Might be nice to elaborate here and say "...because they are expanded by the gnat front-end" or something like that?